docs/Converters.md

```diff
@@ -0,0 +1,16 @@
+# Converters
+
+Converters convert a stream into another type.
+
+### `toPromise`
+> Convert a stream into a promise
+
```

Comment:
```suggestion
Example Uses [from](./Creators.md#from)
```

Reply:
That example would be taking a promise and turning it into a stream.
`toPromise` takes an already existing PureStream and turns it into a promise.

Reply:
Right, I was just pointing out that the example uses another function from the library, and providing a quick link to it.

Reply:
Ohhh I see what you mean! Great idea!